src/app/shared/enum/applications.ts

```diff
@@ -1,4 +1,4 @@
-export enum ApplicationStatus {
+export enum Statuses {
   Pending = 'Pending',
   AcceptedForSelection = 'AcceptedForSelection',
   Approved = 'Approved',
```

Comment:
cause it's not application status anymore so it should be moved to another file (probably the new one)

Reply:
also this enum also has to be refactored  
![image](https://user-images.githubusercontent.com/27493959/198263006-33ff16a4-3482-4f4a-a7c2-4ddf9cb7e921.png)